rxfingerprint/src/main/java/com/mtramin/rxfingerprint/RxFingerprint.java

```diff
@@ -189,6 +189,7 @@ public static boolean isAvailable(@NonNull Context context) {
      *
      * @param context a context
      * @return {@code true} if fingerprint authentication is unavailable
+     * @deprecated redundant method! use {@link RxFingerprint#isAvailable(Context)} instead
```

Comment:
Not really redundant, more for leaving the choice to the developer if he wants to have an `if` with a positive or negative method for checking if Fingerprint authentication is available.